lib/android-helpers.js

```diff
@@ -13,6 +13,7 @@ import {
 } from './unlock-helpers';
 import { EOL } from 'os';
 import semver from 'semver';
+import { validateCaps } from 'appium-base-driver';
```

Comment:
The minimum base driver version in package.json should be updated for this to work

Reply:
updated